doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -31,21 +31,28 @@ Vulkan commands as well; if this is done, then applications that use only
 the core Vulkan commands have no need to use fname:vkGetInstanceProcAddr.
 
 Function pointers to commands that don't operate on a specific instance can:
-be obtained by using this command with pname:instance equal to `NULL`. The
-following commands can: be accessed this way:
+be obtained by using this command with pname:instance equal to `NULL`. Only
+pointers to the following commands can: be obtained this way:
 
   * flink:vkEnumerateInstanceExtensionProperties
   * flink:vkEnumerateInstanceLayerProperties
   * flink:vkCreateInstance
 
-If pname:instance is a valid sname:VkInstance, function pointers to any
-commands that operate on pname:instance or a child of pname:instance can: be
-obtained. The returned function pointer must: only be called with a
-dispatchable object (the first parameter) that is a child of pname:instance.
+If pname:instance is a valid sname:VkInstance, then function pointer to any
+command from any of the following groups can: be obtained:
 
-If pname:pName is not the name of a core Vulkan command, or is an
-extension command for any extension not supported by any available layer or
-implementation, then fname:vkGetInstanceProcAddr will return `NULL`.
+  * core Vulkan commands
+  * commands of instance extensions enabled for pname:instance
+  * commands of device extensions enabled for a valid sname:VkDevice child
```

Comment:
This is not correct.
Right now, the behavior for GetInstanceProcAddr when given a device function name implemented by the system (e.g. can live in the ICD, loader or layer) is to return a function pointer to trampoline code. The extension still needs to be enabled to function, but app would have a valid function pointer.

That was the intent of:
> If pname:pName is not the name of a core Vulkan command, or is an extension command for any extension not supported by any available layer or implementation, then fname:vkGetInstanceProcAddr will return `NULL`.

Maybe a table would be a more appropriate explanation.

sname:VkInstance | pname:pName | return value
------------ | ------------- | ------------
NULL | flink:vkEnumerateInstanceExtensionProperties | fp
NULL | flnk:vkEnumerateInstanceLayerProperties | fp
NULL | flink:vkCreateInstance | fp
NULL | * (any pname:pName not covered above) | NULL
instance | core Vulkan command | fp
instance | enabled instance extension commands for pname:instance | fp
instance | available device extension commands for pname:instance | fp
instance | * (any pname:pName not covered above) | NULL
invalid instance | * | undefined